docs/setup-cluster/k8s/internal-task-gateway.rst

```diff
@@ -15,13 +15,16 @@ refer to the sections below to see the configuration changes and controller requ
 .. warning::
 
    This feature exposes Determined tasks to the outside world. Please ensure that you have the
-   necessary security measures in place to limit access to the exposed tasks. This could include
-   setting up a firewall, using a VPN, IP whitelisting, or other security measures.
+   necessary security measures in place to both limit access to the exposed tasks, and secure the
+   communication between the external cluster and the main one. This could include setting up a
+   firewall, using a VPN, IP white-listing, K8s Network Policies, native cloud solutions, or other
+   security measures. Some Determined tasks including JupyterLab Notebooks and shells already have
+   secure transport built-in.
 
 Limitations:
 
--  Exposing proxies in multi-slot jobs is not supported. Currently this only includes experiments
-   running distributed training that want to manually expose proxies to the outside world.
+-  Proxying to multi-node trials is not supported: Exposing proxies in multi-slot jobs is not supported.
```

Comment:
```suggestion
-  Proxying to multi-node trials is not supported
```

I think a notebook that requests 2 gpus would count as a "multi-slot" job

we do support that case